website/files/models/base.py

```diff
@@ -12,7 +12,6 @@
 from modularodm.exceptions import NoResultsFound
 from dateutil.parser import parse as parse_date
 
-from api.base.utils import absolute_reverse
```

Comment:
Why'd we stop using `absolute_reverse` in all of these places (but not all places)?